docs/en_US/server_dialog.rst

```diff
@@ -205,6 +205,8 @@ Use the fields in the *Advanced* tab to configure a connection:
   command will be used as the SQL password. This may be useful when the password
   should be generated as a transient authorization token instead of providing a
   password when connecting in `PAM authentication <https://www.postgresql.org/docs/current/auth-pam.html>`_ scenarios.
+  You can pass server hostname, port, and DB username to the password exec command as variable by providing placeholders 
+  like ``%DB_HOSTNAME%``, ``%DB_PORT%`` and ``%DB_USERNAME%`` which will be replaced with the server connection information.
```

Comment:
I think `DB_` is redundant and can be removed. It can be same as what we have to title config. Plus, its server hostname, not db hostname.
Please add one example as well.